addons/test_mail/tests/test_message_post.py

```diff
@@ -36,6 +41,27 @@ def test_message_post_return(self):
         messageId = call_kw(self.env['mail.channel'], 'message_post', [test_channel.id], {'body': 'test'})
         self.assertTrue(isinstance(messageId, int))
 
+    @users('employee')
+    def test_mail_add_signature(self):
```

Comment:
Maybe "test_notify_add_signature" to be aligned with the test_notify_prepare_template_context_company_value naming ?